docs/user-guide/modding/advanced/networking.md

```diff
@@ -0,0 +1,377 @@
+# Custom Networking
+
+>**Note:** This is not a tutorial on how to use Unity's [Netcode for GameObjects](https://docs-multiplayer.unity3d.com/netcode/current/tutorials/get-started-ngo/) RPCs and Network Variables.  
+>This is only meant to be used to understand <i>how</i> to implement custom networking into the game.
+
+## Preface
+
+This tutorial requires using [@EvaisaDev](https://github.com/EvaisaDev/)'s [UnityNetcodeWeaver](https://github.com/EvaisaDev/UnityNetcodeWeaver) (Thank you very much Evaisa!). This tutorial will only go into the basics of using this tool; if you run into any issues, ask in the [NetcodePatcher Forum Post](https://discord.com/channels/1169792572382773318/1175504315029389343) on the [Unofficial Lethal Company Community Discord](https://discord.gg/nYcQFEpXfU).
+
+### Why use Unity Netcode Weaver?
+
+Unity Netcode Weaver replicates the IL Post Processing Unity performs when compiling code utilizing Netcode for GameObjects package. Essentially, this allows you to create and use RPCs, Network Variables, etc.
```

Comment:
I might go into slightly more detail *what* this IL post processing step looks like when compiled, since people often come across the autogenerated code and don't know

Could be worth including a screenshot example

Reply:
Decided to add an example ClientRpc being processed